app/controllers/concerns/mentions.rb

```diff
@@ -0,0 +1,9 @@
+module Mentions
+  extend ActiveSupport::Concern
+
+  protected
+
+  def find_mentionable_users
```

Comment:
can we make this a before_filter here for the relevant actions (e.g. `#show`)?

That way we skip cluttering the controller code.